src/sentry/south_migrations/0514_backfill_attachment_group_id.py

```diff
@@ -0,0 +1,1534 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models, transaction
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+class Migration(DataMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = True
+
+    def forwards(self, orm):
+        db.commit_transaction()
+        try:
+            self._forwards(orm)
+        except Exception:
+            db.start_transaction()
+            raise
+        db.start_transaction()
```

Comment:
finally ?

Reply:
Hmmm, I don't want to change this autogenerated stuff, this seems to be present in the other data migrations I've looked at.

Reply:
ok, if it was autogenerated ignore my comment